app/packs/src/apps/mydb/elements/details/samples/qcTab/components/substance/BlockCnmr.js

```diff
@@ -89,23 +88,18 @@ const BlockCnmr = ({ ansCnmr }) => {
             {iconByMargin(ansOwn, 0)}
           </span>
         </div>
-        <Panel
-          className="qc-detail-panel"
-          id="qc-detail-panel-cnmr"
-          defaultExpanded={false}
-        >
-          <Panel.Heading>
-            <Panel.Title className="qc-detail-panel-title" toggle>
+
```

Comment:
Shouldn't this be a card? It's only a single item.

Reply:
If you want the header to collapse and expand the body on click. I would use a single item accordion. It's a pattern that is somewhat special within chemotion, so I think repurposing the accordion component for this is more effective than to invent an interactive variant of the card component.

Reply:
oh ok, I did miss the collapsed part.